WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
Should the `tax_query` flag be respected indefinitely, throwing a warning seems like an unnecessary nag for people using the current flag to me.

If throwing a warning is considered best practice in a case like this (I haven’t found a precedent for this in this repo) I’d be happy to add that.

(For the time being I just combined the two ifs into one).